tests/simple/testdata/timestamps.textproto

```diff
@@ -83,11 +98,26 @@ section {
     expr: "timestamp('2009-02-13T23:31:30Z').getDate('Australia/Sydney')"
     value: { int64_value: 14 }
   }
+  test {
+    name: "getDayOfMonth"
+    expr: "timestamp('2009-02-13T23:31:30Z').getDayOfMonth('US/Central')"
+    value: { int64_value: 12 }
+  }
   test {
     name: "getDayOfMonth"
     expr: "timestamp('2009-02-13T23:31:30Z').getDayOfMonth('+11:00')"
     value: { int64_value: 13 }
   }
+  test {
+    name: "getDay"
+    expr: "timestamp('2009-02-13T02:00:00Z').getDayOfMonth('-2:30')"
+    value: { int64_value: 11 }
+  }
+  test {
+    name: "getDay"
```

Comment:
Test names should be unique within the section.  Also, should be more descriptive, so each "getDayOfMonth" test describes its variation, like "getDayOfMonth_pos", "_neg", and "_name".